rerun_notebook/callbacks.ipynb

```diff
@@ -33,32 +26,45 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Log some data\n",
+    "# To show something interesting, we first need to log some data.\n",
     "rr.init(\"rerun_example_callbacks\")\n",
     "\n",
+    "# This will output a cube grid of points. We'll be able to select\n",
+    "# either the entire grid, or individual points.\n",
     "STEPS = 100\n",
     "twists = math.pi * np.sin(np.linspace(0, math.tau, STEPS)) / 4\n",
     "for t in range(STEPS):\n",
     "    rr.set_time(\"step\", sequence=t)\n",
     "    cube = build_color_grid(10, 10, 10, twist=twists[t])\n",
     "    rr.log(\"cube\", rr.Points3D(cube.positions, colors=cube.colors, radii=0.5))\n",
     "\n",
+    "rr.send_blueprint(rrb.Blueprint(rrb.Spatial3DView(name=\"Everything\")))\n",
     "\n",
-    "# Use callbacks to track the current time and active selection\n",
+    "\n",
+    "# To register callbacks, first inherit from the `ViewerCallbacks` base class,\n",
+    "# and override the event methods you are interested in.\n",
     "class Example(ViewerCallbacks):\n",
     "    def __init__(self) -> None:\n",
+    "        # Because this is a class, we can store whatever state we need in it.\n",
     "        self.selection = \"\"\n",
     "        self.time = \"\"\n",
     "        self.timeline = \"\"\n",
     "\n",
+    "        # Create a Viewer instance, and register ourselves to it.\n",
+    "        # The same callbacks may be registered to multiple Viewers at a time,\n",
+    "        # but there is currently no way to know which Viewer the events are coming from.\n",
```

Comment:
oh interesting limitation. wasn't aware. But I guess that's easy to work around by registering different callbacks to different viewer instances?